services/upload/src/test/java/app/coronawarn/server/services/federation/upload/payload/helper/FakePrivateKeyResource.java

```diff
@@ -0,0 +1,36 @@
+package app.coronawarn.server.services.federation.upload.payload.helper;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import java.io.IOException;
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+
+public class FakePrivateKeyResource {
+
+  private static String TEST_PRIVATE_KEY = "-----BEGIN EC PRIVATE KEY-----\n"
```

Comment:
Just a suggestion, maybe this could be moved in test files under src/test/resources